Utilities/bootstrap

```diff
@@ -815,7 +815,7 @@ def get_swiftpm_flags(args):
 
     build_target = get_build_target(args)
     cross_compile_hosts = args.cross_compile_hosts
-    if build_target == 'x86_64-apple-macosx' and "macosx-arm64" in cross_compile_hosts:
+    if cross_compile_hosts and re.match('macosx-', cross_compile_hosts):
```

Comment:
Rather than check if `cross_compile_hosts` is defined on every line, check it once at the top and then run all these other checks only if it is.

Reply:
Integrated this and Boris' feedbacks in 90c5645c0bbff01c87d6fdce1b0197b4c3e33458